### DISCUSSION TITLE
Possible to run podman and containers from an external drive on mac os?

### DISCUSSION THREAD

**Discussion by anarchy89** [containers/podman#17590] [Q&A]
Is it possible to store and run podman containers from an external drive on mac m1? I only have 256gb of storage and use an external APFS SSD.

**Comment by erlangparasu** [comment]
Any solution?

**Comment by baude** [comment]
podman vm images are stored in `/.local/share/containers/podman/machine/` ... if you were to mount some storage there or one level down which would include the provider, `qemu` (podman4) or `applehv` (podman5), i would think this would work.

the default podman machine is also made out to be 100GB, even though it should be sparse in Podman 5.  You could reduce the size with during `podman machine init`
